Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -484,7 +484,7 @@ def merge
             update_or_add = "update"
             if args.keep_old?
               mismatches = []
-              bottle_block_contents = s[/  bottle do(.+?)end\n/m, 1]
+              bottle_block_contents = s.inreplace_string[/  bottle do(.+?)end\n/m, 1]
```

Comment:
Would it make sense to add the same number of spaces (2) before `end`?... just in case...

Reply:
Umm, it wasn't an issue before I think